test/test_state.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+import time
+
+import dockerblade
+
+@pytest.mark.parametrize('sut', ['turtlebot3-ros2'], indirect=True)
+def test_state_publishers(sut):
+    sut.ros2.launch_manager.launch('simple.launch.py', package='launch')
```

Comment:
`launch` is the name of a package?
Also, this call should be:
```
sut.ros2.launch(....)
```
(we don't need the extra stuff :-))